packages/tracing/src/constants.ts

```diff
@@ -2,4 +2,9 @@
 // Readonly type should enforce that this is not mutated.
 export const FINISH_REASON_TAG = 'finishReason';
 
-export const IDLE_TRANSACTION_FINISH_REASONS = ['heartbeatFailed', 'idleTimeout', 'documentHidden'] as const;
+export const IDLE_TRANSACTION_FINISH_REASONS = [
+  'heartbeatFailed',
+  'idleTimeout',
+  'documentHidden',
+  'finalTimeout',
```

Comment:
given that this shows up in the UI, and I as a user of sentry would probably not understand what this means, does it make more sense to name this something that indicates it's a forced timeout due to thrashing? hard to come up with a concise name but maybe something like `thrashingTimeout` or `forcedThrasingTimeout`? (`idleTimeout` was a similarly confusing name to me, it's not obvious that's the sort of "intended finish reason" for normal transactions, but realize that probably can't be changed anymore)

Reply:
Good catch.

Its behaviour seems like as the [`maxTransactionDuration`](https://docs.sentry.io/platforms/javascript/guides/vue/performance/instrumentation/automatic-instrumentation/#maxtransactionduration) configuration.

Reply:
does it make sense to re-use that configuration option here then (as opposed to creating a new one), and similarly re-use the "deadline_exceeded" finishReason? or is there value in having them both be separate? I guess a default timeout of 600 seconds is pretty excessive for this particular use case...